### PR TITLE
[D1] Update data-location.md

### PR DESCRIPTION
**Pull request by Kiki-Y123**
To check the region of your database in D1

### CODE REVIEW

File: content/d1/configuration/data-location.md
Comment (elithrar) [inline content/d1/configuration/data-location.md:1]: I would suggest moving this up and retitling to "## Check current region"

This section needs to also talk about how this is the location of the _leader_ (writer) and show a wrangler example, including cross-linking to https://developers.cloudflare.com/workers/wrangler/commands/#info

**Review by elithrar — CHANGES_REQUESTED** [review]

File: content/d1/configuration/data-location.md [hunk 1]
```diff
@@ -60,3 +60,7 @@ The following location hints are currently supported:
 | weur          | Western Europe        |
 | eeur          | Eastern Europe        |
 | apac          | Asia-Pacific          |
+
+## To check the region of your database in D1
```
Comment (Maddy-Cloudflare) [inline content/d1/configuration/data-location.md]:
```suggestion
## Check the region of your database in D1
```

File: content/d1/configuration/data-location.md [hunk 2]
```diff
@@ -60,3 +60,7 @@ The following location hints are currently supported:
...
+
+You can use API to ([get d1 database](/api/operations/cloudflare-d1-get-database)) and check field `running_in_region` in the output.
```
Comment (Maddy-Cloudflare) [inline content/d1/configuration/data-location.md]:
```suggestion
You can use the [D1 REST API](/api/operations/cloudflare-d1-get-database) to get your D1 database and check the field `running_in_region` in the output to view the region of your database in D1.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Comment by KimJ15** [comment]
👋Hi there, we’re going to close this PR due to inactivity and web framework migration

**Pull request closed without merging**